### ISSUE TITLE
process 1.6.0

### ISSUE THREAD

**Issue by curtisalexander** [haskell/process#95]
When will `process 1.6.0` be released on stackage -- either nightly or an LTS release?

I ask because I have been trying to install `stack` on Windows and believe that the error I'm receiving was fixed by this [PR](https://github.com/haskell/process/pull/80).

I am quite new to Haskell and `stack` and thus may be completely misunderstanding how everything fits together.

As an alternative, I can open an issue over at the `stack` repo if this is the wrong place for my question.

**Comment by snoyberg** [comment]
The changes are already released, see the changelog at
https://www.stackage.org/package/process. Because process is a wired in
package, Stackage snapshots don't upgrade automatically. Can you open a
Stack issue requesting that the Windows binaries be built with the newer
process?

I think there may also be some code changes necessary to make this all work.

**Comment by curtisalexander** [comment]
@snoyberg Thank you for the response.  I did as you suggested -- here is the [issue](https://github.com/commercialhaskell/stack/issues/3145).  

As I said in the issue I filed, I'll be glad to troubleshoot further if need be.  If my issue is not related to `process` and version 1.6, then I'll be glad to split my issue into two - incorporating `process` 1.6 into the Windows binaries and a separate issue for the issue I'm facing.
